expr/func_aliasbymetric_test.go

```diff
@@ -0,0 +1,170 @@
+package expr
+
+import (
+	"testing"
+
+	"github.com/grafana/metrictank/api/models"
+)
+
+func TestAliasByMetricNameWithoutPeriods(t *testing.T) {
+	// Metric base same as the metric name
+	veryShortMetric := "veryShort"
+	veryShortBase := veryShortMetric
```

Comment:
why both vars?

Reply:
The extracted base metric is usually different, as seen in the tests below. This one follows the same format of separating the extracted base and the read in metric value so it's easier to compare with.

the comment on above the assignments in each test hopes to clarify the differences.